src/anomalib/deploy/inferencers/openvino_inferencer.py

```diff
@@ -56,7 +56,7 @@
 import numpy as np
 import torch
 from lightning_utilities.core.imports import module_available
-from openvino.runtime.utils.data_helpers.wrappers import OVDict
+from openvino.utils.data_helpers.wrappers import OVDict
```

Comment:
As far as I can see `OVDict` is only used for annotation. Maybe we could have the import something like this 

```python
if TYPE_CHECKING:
    from openvino.utils.data_helpers.wrappers import OVDict

....

def post_process(predictions: "OVDict") -> dict:
```